### DISCUSSION TITLE
Other programs

### DISCUSSION THREAD

**Discussion by stefansundin** [stefansundin/altdrag#127] [Show and tell]
Ever since I started AltDrag, I have maintained a list of other programs that may be useful. I even had [a wiki page back on Google Code](https://web.archive.org/web/20130921160045/http://code.google.com/p/altdrag/wiki/OtherPrograms) (that website brings back a lot of memories 😄 ). Since the move to GitHub, I have hosted it on [the AltDrag webpage](https://stefansundin.github.io/altdrag/doc/other-programs.html) (this now redirects to this discussion thread).

It's always fun to discuss the pros and cons of different programs. So please feel free to post a comment if you find more programs, and let me know if the links are dead and if you've found a better link. For forks of AltDrag itself, please use #128.

Right now this is a 1-1 copy of the current page. I haven't really gone through the list, but may do so in a bit.

---

Here is a list of programs that are similar to AltDrag.

## Open source
- [Taekwindow](http://taekwindow.net/)
- [Win32WM](https://virtuawin.sourceforge.io/?page_id=50), a module for [VirtuaWin](https://virtuawin.sourceforge.io/), but can also be used independently.
- [Win-X-Move](http://winxmove.sourceforge.net/)
- ComfortResize in [ac'tivAid](https://activaid.telgkamp.de/)
- [wmove](http://code.google.com/p/wmove/)
- [KDE Mover-Sizer](https://corz.org/windows/software/accessories/KDE-resizing-moving-for-Windows.php), an AutoHotkey script.
- [AltWindowDrag](https://www.howtogeek.com/howto/windows-vista/get-the-linux-altwindow-drag-functionality-in-windows/), another AutoHotkey script.
- [EasyWindowDrag](https://www.autohotkey.com/docs/scripts/EasyWindowDrag.htm), another AutoHotkey script.
- [NiftyWindows](http://www.enovatic.org/products/niftywindows/introduction/), another AutoHotkey script.
- [MoveInactiveWin](http://www.dcmembers.com/skrommel/download/moveinactivewin/), another AutoHotkey script.
- [LinuxMove](http://wincmd.ru/plugring/LinuxMove.html) (quite buggy)
- [FlashWheel](http://flashwheel.sourceforge.net/) (Scroll inactive windows)
- [SeeThroughWindows](http://www.mobzystems.com/Tools/SeeThroughWindows.aspx) (Transparency toggler)
- [RBTray](http://rbtray.sourceforge.net/) (Minimize to tray by right clicking minimize buttons)

## Freeware
- [AquaSnap](https://www.nurgo-software.com/products/aquasnap) (Snap on move and resize)
- [EasyMove](http://www.chrisnsoft.com/download)
- [Pitaschio](https://www.majorgeeks.com/files/details/pitaschio.html) (Snap on move and resize)
- [allSnap](https://download.cnet.com/AllSnap/3000-2072_4-10120369.html) (Window snapping)
- [WinMover](https://www.majorgeeks.com/files/details/winmover.html) (**Note:** This program didn't work for me and hung my Vista machine when I tried to run it)
- [KatMouse](http://ehiti.de/katmouse/) (Scroll inactive windows)
- [WizMouse](https://antibody-software.com/web/software/software/wizmouse-makes-your-mouse-wheel-work-on-the-window-under-the-mouse/) (Scroll inactive windows)
- [Mouse Hunter](http://amlpages.com/mousehunter.shtml) (Scroll inactive windows)
- [Volumouse](http://www.nirsoft.net/utils/volumouse.html) (Change volume with scroll wheel)
- [4t Tray Minimizer](http://www.4t-niagara.com/tray.html) (Adds minimize to tray, roll up/down, toggle always on top, and transparent buttons to titlebars)
- [PowerMenu](https://www.softpedia.com/get/System/OS-Enhancements/PowerMenu.shtml) (Adds minimize to tray, transparency and toggle always on top to sysmenus. Only works on 32-bit windows and does not work properly on Windows 7)
- [X-Mouse Button Control](http://www.highrez.co.uk/downloads/XMouseButtonControl.htm)
- [AltMove](http://www.deskex.com/altmove/) (**Note:** I could not get it working in Windows 8.1)
- [FeeWhee](http://www.nattyware.com/feewhee.php) (Resize windows by scrolling the titlebar, hold the Shift key to change transparency)

## Related
- [MouseImp](http://www.mouseimp.com/)
- [DragLock](http://www.dcmembers.com/skrommel/download/draglock/)
- [True X-Mouse](http://fy.chalmers.se/~appro/nt/TXMouse/)
- [WinRoll](https://www.softpedia.com/get/Desktop-Enhancements/Other-Desktop-Enhancements/WinRoll.shtml)
- [Edgeless 2](https://www.softpedia.com/get/Desktop-Enhancements/Other-Desktop-Enhancements/Edgeless.shtml)
- [7+ Taskbar Tweaker](https://rammichael.com/7-taskbar-tweaker)

## Commercial
- [Actual Multiple Monitors](https://www.actualtools.com/multiplemonitors/) (Window snapping, $25 to buy)

## Mac OS X
- [Flexiglass](https://nulana.com/flexiglass/) (15-day sharenagware, $10 to buy, still works when expired)
- [BetterTouchTool](https://folivora.ai/)

## Other
- [List of window management programs](https://www.techsupportalert.com/content/best-free-window-manager-resizer-arranger.htm)

**Comment by plivxo** [comment]
DesktopOK (https://www.softwareok.com/?seite=Freeware/DesktopOK) does many things but has an AltDrag function which works fine for me in Windows 11 with non-standard DPI settings.

**Comment by giveo-logan** [comment]
https://github.com/RamonUnch/AltSnap is a continuation of AltDrag with everything fixed for modern operating systems including DPI.

**Reply by stefansundin** [reply]
Forks are discussed in #128.
